distros/Dockerfile.alpine

```diff
@@ -1,25 +0,0 @@
-FROM alpine:3.12
```

Comment:
Why is this file deleted, and not replaced with the main directory Dockerfile?

Reply:
Because it is a duplicate file. The same file is in the main directory and the distros directory. If you don't like this, I'll put the Dockerfile from the main directory also in distros directory.

Reply:
Nah, delete it. I'll go over the Dockerfiles in the `distros` directory later.